src/_data/sites/mindos.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://mindos.in/",
+    "name": "MindOS",
+    "description": "A 10 year project to create an Operating System for the mind",
+    "twitter": "inmindos",
+    "authoredBy": ["binnyva"],
```

Comment:
Is this another Twitter username?

https://github.com/11ty/11ty-website/issues/new?assignees=&labels=new-site%21&template=built-with-eleventy.md&title=I+built+a+site+with+Eleventy%21
* `authoredBy`: An array of Twitter usernames of the site’s authors. Supplements the `twitter` entry. (Optional)

Reply:
Yeah, there is a separate twitter account I created for this project - twitter.com/inmindos . But the author is @binnyva.